python/cudf/cudf/_lib/pylibcudf/groupby.pyx

```diff
@@ -28,6 +29,12 @@ from .utils cimport _as_vector
 cdef class GroupByRequest:
     """A request for a groupby aggregation or scan.
 
+    This class is functionally polymorphic and can represent either an
+    aggregation or a scan depending on the algorithm it is used with. For
+    details on the libcudf types it converts to, see
+    :cpp:class:`cudf::groupby::aggregation_request` and
+    :cpp:class:`cudf::groupby::scan_request`.
```

Comment:
Hmm yeah it may be. In the interest of speed I'm going to merge this PR now, but I do think all of pylibcudf's documentation deserves a thorough review before we actually expose the API. I'll make a note that we should check on this docstring in particular when doing that.